tests/unit/mixins/token-adapter-test.js

```diff
@@ -0,0 +1,11 @@
+import EmberObject from '@ember/object';
+import TokenAdapterMixin from 'ember-simple-auth-token/mixins/token-adapter';
+import { module, test } from 'qunit';
+
+module('Unit | Mixin | token-adapter', function() {
+  test('it works', function (assert) {
+    const TokenAdapterObject = EmberObject.extend(TokenAdapterMixin);
+    const subject = TokenAdapterObject.create();
+    assert.ok(subject);
+  });
```

Comment:
could we add a simple test for the `headers` computed property?

Reply:
Will do

Reply:
I have having some difficulties. Have you every written unit tests for a computed property in a mixin?

Reply:
I gave up